Explorer/Assets/DCL/Input/Component/InputMapComponent.cs

```diff
@@ -56,14 +56,15 @@ public InputMapComponent(Kind flags)
         public void BlockInput(Kind kind)
         {
             inputBlockCounters[kind] += 1;
+            ReportHub.Log(ReportCategory.INPUT, $"Block {kind} - {inputBlockCounters[kind]}");
```

Comment:
I left it because all the times I needed to debug input I had to re-add these and as our logs are stripped from prod builds and these are only visible if we enable the Report Category. So they shouldn't be a burden.